service/create/service.go

```diff
@@ -180,6 +184,15 @@ func (s *Service) Boot() {
 					s.awsConfig.Region = cluster.Spec.AWS.Region
 					awsSession, ec2Client := awsutil.NewClient(s.awsConfig)
 
+					// Create keypair
+					keyPairName, err := s.keyPair(keyPairInput{
+						ec2Client:   ec2Client,
+						clusterName: cluster.Name,
+					})
+					if err != nil {
+						s.logger.Log("error", fmt.Sprintf("could not create keypair: %s", err))
```

Comment:
What happens when we cannot create the key pair? Do we just go ahead? That feels wrong. There should probably retries and stuff?